resource/decker/support/plugins/live-captioning/live-captioning.js

```diff
@@ -65,9 +65,8 @@ function getLanguageCode(string) {
  * Plugin class to be registered so the activation-(decker-)button is at the right place.
  */
 class LiveCaptioning {
-  constructor() {
-    this.reveal = undefined;
-    this.id = "live-captioning";
+  constructor(reveal) {
```

Comment:
why is this plugin involved in this branch?

Reply:
It seems I forgot to add these files to the speaker notes fix branch and it got added in an add -u later in this branch. Sorry for that, we can cherry pick this out into develop.